tests/security/eal4/syscall_thrasher.pm

```diff
@@ -12,12 +12,17 @@ use strict;
 use warnings;
 use testapi;
 use utils;
+use Utils::Architectures 'is_s390x';
 
 my $log_file = '/tmp/syscalls_output.log';
 
 sub run {
     my ($self) = shift;
 
+    # Skip on s390x with encrypted disk due to timeouts
+    # poo#176454
+    return if get_var('ENCRYPT') and is_s390x;
```

Comment:
I think it would be better to make it visible directly in the web ui why we skipped the test, like

`record_info('SKIPPING TEST', "Skipping test due to poo#176454.");`